admin/views/redirects.php

```diff
@@ -0,0 +1,237 @@
+										<span class="select2-selection__rendered" id="select2-wpseo_redirects_type-container" role="textbox" aria-readonly="true" title="301 Moved Permanently">
+											<?php
+											\esc_html_e( '301 Moved Permanently', 'wordpress-seo' )
+											?>
+										</span>
+										<span class="select2-selection__arrow" role="presentation">
+											<b role="presentation"></b>
+										</span>
+									</span>
+								</span>
+								<span class="dropdown-wrapper" aria-hidden="true"></span>
+							</span>
+						</div>
+
+						<p class="label desc description wpseo-redirect-clear">
+							<?php
+							printf(
+								/* translators: 1: opens a link. 2: closes the link. */
+								esc_html__( 'The redirect type is the HTTP response code sent to the browser telling the browser what type of redirect is served. %1$sLearn more about redirect types%2$s.', 'wordpress-seo' ),
+								'<a href="#" target="_blank">',
+								'</a>'
+							);
+							?>
+						<div class="redirect_form_row" id="row-wpseo_redirects_origin">
+							<label class="textinput" for="wpseo_redirects_origin">
+								<span class="title">
+									<?php
+									\esc_html_e( 'Old URL', 'wordpress-seo' )
+									?>
+								</span>
+							</label>
+							<input type="text" class="textinput" name="wpseo_redirects_origin" id="wpseo_redirects_origin" value="" tabindex="-1">
+						</div>
+						<br class="clear">
+
+						<div class="redirect_form_row wpseo_redirect_target_holder" id="row-wpseo_redirects_target">
+							<label class="textinput" for="wpseo_redirects_target">
+								<span class="title">
+									<?php
+									\esc_html_e( 'URL', 'wordpress-seo' )
+									?>
+								</span>
+							</label>
+							<input type="text" class="textinput" name="wpseo_redirects_target" id="wpseo_redirects_target" value="" tabindex="-1">
+						</div>
+						<br class="clear">
+
+						<button type="button" class="button button-primary" tabindex="-1">
+							<?php
+							\esc_html_e( 'Add Redirect', 'wordpress-seo' )
+							?>
+						</button>
+					</div>
+				</form>
+				<p class="desc">&nbsp;</p>
+				<form id="plain" class="wpseo-redirects-table-form" method="post" action="">
+					<input type="hidden" class="wpseo_redirects_ajax_nonce" name="wpseo_redirects_ajax_nonce" value="6ccb86df42">
+					<input type="hidden" id="_wpnonce" name="_wpnonce" value="4b02cca185">
+					<input type="hidden" name="_wp_http_referer" value="/wp-admin/admin.php?page=wpseo_redirects">	<div class="tablenav top">
+
+					<div class="alignleft actions">
+						<select name="redirect-type" id="filter-by-redirect" tabindex="-1">
+							<option selected="selected" value="0">
+								<?php
+								\esc_html_e( 'All redirect types', 'wordpress-seo' )
+								?>
+							</option>
+						</select>
+						<input type="submit" name="filter_action" id="post-query-submit" class="button" value="<?php \esc_attr_e( 'Filter', 'wordpress-seo' ); ?>" tabindex="-1">
```

Comment:
Minor thing, but maybe better to have that as `type="button"` so as to not cause reload after clicked? 

One should manually delete the Unlock with Premium div to do so, but still it's doable

Reply:
Makes sense, fixed